docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1964,6 +1964,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/core/maxPageSize`|integer|Sets the maximum page size for search and history request results. If a user-specified `_count` parameter value exceeds the maximum page size, then a warning is logged and the maximum page size will be used.|
 |`fhirServer/core/maxPageIncludeCount`|integer|Sets the maximum number of 'include' resources allowed per page for search and history request results. If the number of 'include' resources returned for a page of results from a search or history request will exceed the maximum number of 'include' resources allowed per page, then an error will be returned in the request results.|
 |`fhirServer/core/capabilitiesUrl`|string|The URL that is embedded in the default Capabilities statement|
+|`fhirServer/validation/failFast`|boolean|Indicates whether validation should fail fast on create and update interactions|
```

Comment:
do we have plans for other properties under `fhirServer/validation`?  if not, it might make sense just to stick it under `fhirServer/core`.  we already have a couple validation-related properties there.

Reply:
Per our discussion, yes we may have additional properties when we add support for extending the validator.